docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -389,7 +389,7 @@
     }
    ],
    "source": [
-    "display(pd.DataFrame(output_data[\"line\"][\"p_from\"][0]))"
+    "display(pd.DataFrame(output_data[ComponentType.line][\"p_from\"][0]))"
```

Comment:
So we are now only doing this at component level, not attribute level ;)

Reply:
@Jerry-Jinfeng-Guo Yes. Only custom datatypes have been introduced for `DataType` and `ComponentType`.

Reply:
@mgovers @Jerry-Jinfeng-Guo Since this design discussion is probably outside the scope of this ticket, I will resolve the conversation and open a Jira ticket in the backlog for future possible discussion and implementation.